frontend/css/map.css

```diff
@@ -0,0 +1,105 @@
+body {
+    margin: 0 0 0 0;
+    padding: 0 0 0 0;
+    overflow-y: hidden;
+}
+/* HEADER BOX */
+header {
+    background-color: rgb(167, 60, 113);
+    filter: drop-shadow(0px 10px 10px rgba(0,0,0,.15));
+    width: 100vw;
+    height: 11vh;
+    position: absolute;
+    margin: 0 0 0 0;
+    z-index: 2;
+}
+#bus-name {
+    color: white;
+    margin: 0 0 0 25%;
+    padding: 0 0 0 0;
+    font-family: 'Lato';
+    font-size: 22px;
+    height: 11vh;
+    line-height: 11vh;
+    text-align: center;
+    width: 50vw;
+    filter: none;
+    position: absolute;
+    z-index: 3;
+}
```

Comment:
b/c the `<h1>` w/the "bus-name" id is now in a "flex box" we're going to need to make some changes to this class as well so that it plays, below are my suggestions, but feel free to edit this differently if u'd like:
```css
#bus-name {
    color: white;
    /* margin: 0 0 0 25%;  << get rid of this */
    marign: 0; /* && make it 0 instead */
    padding: 0 0 0 0;
    font-family: 'Lato';
    font-size: 22px;
    height: 11vh;
    line-height: 11vh;
    text-align: center;
    /* width: 50vw;  << get rid of this */
    display: inline-block; /* && make it a inline-block element instead */
    filter: none;
    /* position: absolute; << get rid of this */
    /* z-index: 3; << && get rid of this */
}
```